src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+const successHandler = (message) => {
+  const messageDiv = document.createElement('div');
+
+  messageDiv.className = 'message';
+  messageDiv.textContent = message;
+  document.body.appendChild(messageDiv);
+};
+
+const errorHandler = (error) => {
+  const messageDiv = document.createElement('div');
+
+  messageDiv.className = 'message error-message';
+  messageDiv.textContent = error.message;
```

Comment:
These functions perform the same actions with different class names and text content. I think we could create a utility function to keep the code DRY.